### ISSUE TITLE
Add Support for React 15

### ISSUE THREAD

**Issue by joncursi** [meteor/chromatic#7]
Peer requirements are not satisfied if react is upgraded to 15.x

<img width="406" alt="screen shot 2016-04-10 at 9 14 54 am" src="https://cloud.githubusercontent.com/assets/4974609/14410504/07ea0488-feff-11e5-97b7-8b991fd4ced5.png">

**Comment by gerwinbrunner** [comment]
Is there an ETA for this fix? I'm also using react 15 and ran into the same issue.

**Comment by joncursi** [comment]
@gerwinbrunner I started using [React Storybook](https://github.com/kadirahq/react-storybook) to fill the gap. I might come back to chromatic whenever this package gets a React 15 update, but I know MDG is focused on other things right now (Apollo) so it might be a while.

**Comment by ffxsam** [comment]
@gerwinbrunner With Storybook, how are you dealing with:

1. Absolute paths in your imports (`import XYZ from '/some/path/XYZ'`)
2. Components imported by Storybook that import Meteor packages (this breaks for me)

**Comment by serkandurusoy** [comment]
We keep hitting walls with storybook because it is too disjointed with meteor but a meteor project in essence has lots of meteor dependencies. Furthermore, having two different build systems is also getting in the way.

I wish chromatic dependencies were just updated so that we would be able to use this instead.

**Comment by ffxsam** [comment]
Yeah, with Storybook you _really_ have to have 100% pure React components if you want to add them into Storybook. Any Meteor packages, and it chokes. I've gotten into the habit of trying to put all Meteor specific stuff in the container component, and then the child component below that is 100% pure React.

**Comment by serkandurusoy** [comment]
Yep, but the big downside is, that makes it next to impossible to test nested components which only make sense when stitched together. There are cases where the container gets nested inside a parent (scoped data etc) and it really forces you to pull all data fetching to outer components, which may not always be desirable.

All in all, I don't like it when "a mere utility forces design decisions" on me.

**Comment by ffxsam** [comment]
I agree. Meteor is its own ecosystem, and apparently React Storybook wasn't really created with Meteor in mind. Hopefully development on Chromatic picks up again soon, after Apollo is done.

**Comment by loredanacirstea** [comment]
I built upon the `npm` branch and added partial support for Meteor 1.3 and React 15 + some nice features:

- Dynamic exploration, design and testing - dat.GUI interface for changing React props
- Just copy-paste the new JSON settings into your React component
- Supports HTML and SVG
- In work: having your stylesheet classes at your GUI fingertips

Check:
https://github.com/oro8oro/chromatic
Short demo: https://www.youtube.com/edit?video_id=DVFj2A6789Y

**Comment by Gywem** [comment]
React 15.x is currently supported, however I still get the warning. I have included `mdg:chromatic` along with all component packages.

```
WARNING: npm peer requirements not installed:
 - react@15.1.0 installed, react@0.14.x needed
 - react-dom@15.1.0 installed, react-dom@0.14.x needed
 - react-addons-transition-group@15.1.0 installed, react-addons-transition-group@0.14.x needed
 - react-addons-css-transition-group@15.1.0 installed, react-addons-css-transition-group@0.14.x needed
 - react-addons-linked-state-mixin@15.1.0 installed, react-addons-linked-state-mixin@0.14.x needed
 - react-addons-create-fragment@15.1.0 installed, react-addons-create-fragment@0.14.x needed
 - react-addons-update@15.1.0 installed, react-addons-update@0.14.x needed
 - react-addons-pure-render-mixin@15.1.0 installed, react-addons-pure-render-mixin@0.14.x needed
 - react-addons-test-utils@15.1.0 installed, react-addons-test-utils@0.14.x needed
 - react-addons-perf@15.1.0 installed, react-addons-perf@0.14.x needed

Read more about installing npm peer dependencies:
  http://guide.meteor.com/using-packages.html#peer-npm-dependencies
```

Something weird happens because when I get back to the `0.14` version, then it warns me about the need of the `15` version.
